content/en/docs/community_roles/contributor.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Project Contributor
+hide_readingtime: true
+weight: 20
+---
+
+A Project Contributor plays a pivotal role in shaping the project through code, documentation, or design. Whether fixing bugs, implementing features, or enhancing the user experience, contributors bring fresh perspectives and energy. Their work supports the project’s evolution, bridging the gap between the project's goals and the community's needs.
+
+## How to Become a Contributor
+
+To [contribute](../contributing) to a project, individuals must first interact with _Paul Botsco_ to sign our Contributor License Agreement (CLA), ensuring that their contributions comply with our legal requirements.
```

Comment:
```suggestion
To [contribute]({{< ref "/docs/contributing" >}}) to a project, individuals must first interact with _Paul Botsco_ to sign our Contributor License Agreement (CLA), ensuring that their contributions comply with our legal requirements.
```

Reply:
Will this change also be needed elsewhere?